web/static/js/__tests__/Page-test.jsx

```diff
@@ -1,17 +1,14 @@
-/** @jsx React.DOM */
-
 "use strict"
 
 var React = require('react');
+var TestUtils = require('react-addons-test-utils');
 
 var Page = require('../Page.jsx');
 
 describe('Page', function() {
   it('renders without throwing', function() {
-    var container = document.createElement('div');
-    React.renderComponent(
-      <Page />,
-      container
+    TestUtils.renderIntoDocument(
+      <Page location={{query: ''}} />
```

Comment:
Oh yeah, this is an ugly hack. In the real app, `react-router` supplies that property. I should at least add a comment here.